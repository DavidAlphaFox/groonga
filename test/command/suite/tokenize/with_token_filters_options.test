#@on-error omit
plugin_register token_filters/stem
#@on-error default

tokenize \
  TokenBigram \
  "maintenait" \
  NormalizerAuto \
  --token_filters 'TokenFilterStem("algorithm", "french")'
