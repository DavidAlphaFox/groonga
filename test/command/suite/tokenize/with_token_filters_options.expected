plugin_register token_filters/stem
[[0,0.0,0.0],true]
tokenize   TokenBigram   "maintenait"   NormalizerAuto   --token_filters 'TokenFilterStem("algorithm", "french")'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "mainten",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
